kmir/poetry.lock

```diff
@@ -699,14 +700,14 @@ files = [
 
 [[package]]
 name = "pytest"
-version = "8.2.1"
+version = "8.2.2"
```

Comment:
I am unsure why I am getting an [error on CI](https://github.com/runtimeverification/mir-semantics/actions/runs/9416206155/job/25938882959?pr=362#step:4:710) for version `8.2.0` colliding with `8.2.2` when the old version that is working on CI is `8.2.1`